packages/lambda-xyz/src/__test__/wmts.capability.test.ts

```diff
@@ -58,6 +58,11 @@ o.spec('WmtsCapabilities', () => {
         o(layer?.find('ows:Title')?.textContent).equals('aerial:title');
         o(layer?.find('ows:Identifier')?.textContent).equals('aerial-3857');
 
+        o(layer?.find('Style')?.textContent).equals(
+            V('Style', { isDefault: 'true' }, [V('ows:Title', 'Default Style'), V('ows:Identifier', 'default')])
+                .textContent,
+        );
+
```

Comment:
`textContent` does Not test that the element names are correct. I suggest using `toString`

Reply:
:+1: Okay. I did consider that but saw the prior test was using textContent